pwndbg/commands/ai.py

```diff
@@ -414,8 +465,9 @@ def ai(question, model, temperature, max_tokens, verbose, list_models=False, com
         last_question.pop(0)
         last_answer.pop(0)
 
-    term_width = os.get_terminal_size().columns
-    answer = textwrap.fill(res, term_width, replace_whitespace=False)
+    # term_width = os.get_terminal_size().columns
+    # answer = textwrap.fill(res, term_width, replace_whitespace=False)
+    answer = res
     print(M.success(answer))
```

Comment:
Should we just print `res` and remove the comments?

Reply:
yes. i just couldn't get textwrap to reliably make things pretty.